specs/exec-engine.md

```diff
@@ -64,29 +63,29 @@ fee payments are not registered as internal EVM calls, and thus distinguished be
 These are hardcoded addresses, pointing at pre-deployed proxy contracts.
 The proxies are backed by vault contract deployments, based on `FeeVault`, to route vault funds to L1 securely.
 
-| Vault Name          | Predeploy                                                |
-|---------------------|----------------------------------------------------------|
-| Proposer Fee Vault | [`ProposerFeeVault`](./predeploys.md#ProposerFeeVault) |
-| Base Fee Vault      | [`BaseFeeVault`](./predeploys.md#BaseFeeVault)           |
-| L1 Fee Vault        | [`L1FeeVault`](./predeploys.md#L1FeeVault)               |
+| Vault Name             | Predeploy                                                      |
+|------------------------|----------------------------------------------------------------|
+| Validator Reward Vault | [`ValidatorRewardVault`](./predeploys.md#ValidatorRewardVault) |
+| Protocol Vault         | [`ProtocolVault`](./predeploys.md#ProtocolVault)               |
+| Proposer Reward Vault  | [`ProposerRewardVault`](./predeploys.md#ProposerRewardVault)   |
 
-### Priority fees (Proposer Fee Vault)
+### Transaction Fees
 
-Priority fees follow the [eip-1559] specification, and are collected by the fee-recipient of the L2 block.
-The block fee-recipient (a.k.a. coinbase address) is set to the Proposer Fee Vault address.
+Transaction fees in Kroma are different from [eip-1559] specification.
+The Base Fee is not burned, and there is no distinction between Base Fee and Priority Fee.
+The transaction fee is distributed to two vaults, Validator Reward Vault and Protocol Fee.
+- Validator Reward Vault: `(baseFee + priorityFee) * ValidatorRewardRatio)`
```

Comment:
Please put blank link here!

See `MD032/blanks-around-lists: Lists should be surrounded by blank lines`